homeassistant/components/comelit/strings.json

```diff
@@ -23,11 +23,26 @@
           "pin": "[%key:component::comelit::config::step::reauth_confirm::data_description::pin%]",
           "type": "The type of your Comelit device."
         }
+      },
+      "reconfigure": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "pin": "[%key:common::config_flow::data::pin%]",
+          "type": "[%key:component::comelit::config::step::user::data::type%]"
```

Comment:
Can the type change? Or the port or pin? Reconfigure means we connect to the same device

Reply:
Fo the same device, type not, but port and pin yes of course

Reply:
Removed `type`